modules/logging-rn-5.7.3.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-3_{context}"]
+= Logging 5.7.3
+This release includes link:https://access.redhat.com/errata/RHSA-2023:3998[OpenShift Logging Bug Fix Release 5.7.3].
+
+[id="openshift-logging-5-7-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, when viewing logs within the {product-title} web console, cached files caused the data to not refresh. With this update the bootstrap files are not cached, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4100[LOG-4100])
+
+* Before this update, the Loki Operator reset errors in a way that made identifying configuration problems difficult to troubleshoot. With this update, errors persist until the configuration error is resolved. (link:https://issues.redhat.com/browse/LOG-4156[LOG-4156])
+
+* Before this update,  the LokiStack ruler did not restart caused when the administrator made changes to the `RulerConfig` CR. With this update, the Loki Operator restarts the ruler pods after the `RulerConfig` CR is updated. (link:https://issues.redhat.com/browse/LOG-4161[LOG-4161])
+
+* Before this update, using vector with input match label values containing a `/` character within the `ClusterLogForwarder` CR would cause the collector to terminate unexpectedly. This update resolves the issue by quoting the match label, enabling the collector to start and collect logs. (link:https://issues.redhat.com/browse/LOG-4176[LOG-4176])
+
+* Before this update, the Loki Operator terminated unexpectedly when a `LokiStack` CR defined tenant limits, but not global limits. With this update, the Loki Operator can process `LokiStack` CRs without global limits, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4198[LOG-4198])
+
+* Before this update, Fluentd was unable to send logs to an Elasticsearch cluster when a passphrase applied to the private key provided.
+With this update, Fluentd properly handles passphrase protected private keys when establishing a connection with Elasticsearch. (link:https://issues.redhat.com/browse/LOG-4258[LOG-4258])
+
+* Before this update, clusters with more than 8,000 namespaces caused Elasticsearch to reject queries because the list of namespaces was larger than the `http.max_header_size` setting. With this update, the default value for header size has been increased to 512kb, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4277[LOG-4277])
+
+* Before this update, label values containing a `/` character within the `ClusterLogForwarder` CR would cause the collector to terminate unexpectedly.
+With this update, slashes are replaced with underscores, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4095[LOG-4095])
```

Comment:
This looks like a duplicate (the two Jiras might be duplicates) of the item on a previous line, https://github.com/openshift/openshift-docs/pull/62331/files#diff-ed92149ec7d28da3140ff1b638f5ff1569e61288ccfb06c13780cbd2f907fc5cR16.
However, here's a different solution.
Perhaps talk to the SMEs and possibly combine these two items into one?